src/bandwidth.rs

```diff
@@ -161,8 +161,11 @@ pub trait UpdateBandwidthSinks {
 
 #[cfg(feature = "webrtc")]
 impl UpdateBandwidthSinks for libp2p_webrtc::tokio::Connection {
-    fn update_bandwidth_sinks(self: Pin<&mut Self>, _: &BandwidthSinks) {
-        // TODO: Fetch data from connection via public functions and update sinks
+    fn update_bandwidth_sinks(self: Pin<&mut Self>, sinks: &BandwidthSinks) {
+        sinks.inbound.store(self.total_inbound(), Ordering::Relaxed);
```

Comment:
@thomaseizinger `sinks` is per connection, right? i.e. `store` is fine

Reply:
I believe `sinks` will be across the entire transport. At least I think that is the case in the current code.

Reply:
> @thomaseizinger `sinks` is per connection, right? i.e. `store` is fine

I don't think `store` is not going to be fine. We need to report the total amount of bytes written here. Exposing that from `Connection` is a bit hacky though.

Alternatively, we could only report the bytes written since we last called `total_inbound` (should have a different name then I guess). That would allow us to add the numbers up here.

Reply:
`store` is fine if `connection::total_inbound` returns a total inbound across all connections, which seems completely wrong from API design perspective 😄 

> Alternatively, we could only report the bytes written since we last called total_inbound (should have a different name then I guess). That would allow us to add the numbers up here.

Yep 👍